catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/SceneAdapter.java

```diff
@@ -49,10 +49,11 @@ public void onBindViewHolder(ExtendedVH holder, int position) {
 		ProjectAndSceneScreenshotLoader loader = new ProjectAndSceneScreenshotLoader(thumbnailWidth, thumbnailHeight);
 		Scene item = items.get(position);
 
-		File projectDir = ProjectManager.getInstance().getCurrentProject().getDirectory();
-		holder.title.setText(item.getName());
-
-		loader.loadAndShowScreenshot(projectDir.getName(), item.getDirectory().getName(), false, holder.image);
+		if (ProjectManager.getInstance().getCurrentProject() != null) {
```

Comment:
at this point and in the SceneAdapter the current project must never be null.
what you are doing is just avoiding the Exception, but then the user will see an empty screen if it happens.
the problem is somewhere else, mainly should be when deserializing/loading the project or setting the current project to the loaded one. please try to start debugging from there.